pkg/cmd/task/list.go

```diff
@@ -76,6 +76,13 @@ func printTaskDetails(s *cli.Stream, p cli.Params) error {
 		return err
 	}
 
+	// Check if namespace exists. Return error if namespace specified with -n doesn't exist.
+	_, err = cs.Kube.CoreV1().Namespaces().Get(p.Namespace(), metav1.GetOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: Namespace \"%s\" does not exist. Please enter a valid namespace.\n", p.Namespace())
```

Comment:
I think we need to handle the case of `if the user doesn't have access to that namespace`. In that case, this error message may be misleading.

Reply:
It may be better to just return `err` here as what ever is returned will be displayed to the user. I ran into issues with testing by returning `err` originally since it appears to display the usage as part of the output. I'll take another shot at that approach.

Reply:
This should now display RBAC errors as well with most recent push.